src/components/Charts/ZoneChart.style.js

```diff
@@ -74,16 +75,16 @@ export const ZoneChartWrapper = styled.div`
   }
 
   .ZoneAnnotation--isActive {
-    &.ZoneAnnotation--High rect {
-      fill: ${LEVEL_COLOR.HIGH};
+    &.ZoneAnnotation--2 rect {
```

Comment:
Oooh, yikes.  That is interesting that we end up using the enum value in the class names (good catch)!  I'm not a huge fan, but I assume Pablo is reworking all of this.  So 🤷

Reply:
ya same same-- i think it's an argument to keep the string enums, but idk at least it's clear now what is happening